backend/src/main/java/com/roc/app/referee/general/Referee.java

```diff
@@ -0,0 +1,35 @@
+package com.roc.app.referee.general;
+
+import com.roc.app.referee.licence.RefereeLicence;
+import com.roc.app.user.general.User;
+import jakarta.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+
+@Entity
+@Table(name = "referees")
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class Referee {
+    @Id
+    @Column(name = FieldNames.USER_ID)
+    private Long userId;
```

Comment:
I just noticed missing annotation.
```suggestion
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long userId;
```

Reply:
Is it necessary to add that annotation? Table `referee` contains one field, which is both a primary and foreign key (we can't insert a new referee, if it doesn't exist in `users` table).